### DISCUSSION TITLE
Translation issues

### DISCUSSION THREAD

**Discussion by asdiasx** [GrapesJS/grapesjs#5210] [Q&A] [answered]
Hello everyone! 
I'm currently working with grapesjs version 0.20.3, and I need to translate the options for text alignment (Left, Center, Right, Justify) and also the Border Collapse options (Yes/No) into Portuguese ( on Style Manager / Properties ). 
However, I'm having trouble finding a way to do this in the locale file (pt.js).

Has anyone else encountered the same issue? If so, could you please provide some hints or guidance on how to achieve this?

<img width="296" alt="translate_text_align" src="https://github.com/GrapesJS/grapesjs/assets/13187501/fc56236f-bb0e-4fae-a847-3222e11751e4">

Thank you in advance!

**Comment by artf** [comment, marked as answer]
You can see an example here https://github.com/GrapesJS/grapesjs/blob/f16638a0f3da126dd68af069975b915f0569fb98/src/i18n/locale/en.js#L128-L134

**Reply by asdiasx** [reply]
Thank you, @artf, for your help with setting translations! I really appreciate it. 
Congratulations on your amazing application! Best regards.

**Comment by PrAnacleto** [comment]
@asdiasx I would like to know how did you change the language to portuguese(pt).
![Screenshot 2023-07-10 175512](https://github.com/GrapesJS/grapesjs/assets/50595168/32c29aee-2ab8-4b53-b52e-f82fdbe1fbdc)
![Screenshot 2023-07-10 175550](https://github.com/GrapesJS/grapesjs/assets/50595168/ed6b5362-e654-41ce-8e67-591aa575d9a7)

please help me.
